piaf.opam

```diff
@@ -15,6 +16,7 @@ depends: [
   "angstrom"
   "faraday"
   "bigstringaf" {>= "0.5.0"}
+  "ezgzip"
```

Comment:
this is not a dependency of Piaf. It's only used in carl `bin/carl.ml`. It's omitted from the opam file because that file isn't part of the piaf package.

Reply:
Ok, I can move this install away from the opam file if you are still interested in the PR